x-pack/plugins/apm/server/routes/alerts/rule_types/utils/get_interval_in_seconds.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const intervalUnits = ['y', 'M', 'w', 'd', 'h', 'm', 's', 'ms'];
+const INTERVAL_STRING_RE = new RegExp(
+  '^([0-9\\.]*)\\s*(' + intervalUnits.join('|') + ')$'
+);
+
+interface UnitsToSeconds {
+  [unit: string]: number;
+}
+
+const units: UnitsToSeconds = {
+  ms: 0.001,
+  s: 1,
+  m: 60,
+  h: 3600,
+  d: 86400,
+  w: 86400 * 7,
+  M: 86400 * 30,
+  y: 86400 * 356,
+};
+
+const NUM_BUCKETS = 5;
```

Comment:
This should be imported from here https://github.com/elastic/kibana/pull/158439/files#diff-4fb21ef6e1d21c7026ceab628776c11b01770d418dcad6b594de9217bd241d1cR21

Reply:
I moved this to common folder to import from both public and server.

```
export const NUM_BUCKETS_FOR_PREVIEW_CHART = 5;
```